installing/installing-mirroring-installation-images.adoc

```diff
@@ -0,0 +1,84 @@
+[id="installing-mirroring-installation-images"]
+= Mirroring images for a disconnected installation
+include::modules/common-attributes.adoc[]
+:context: installing-mirroring-installation-images
+
+toc::[]
+
+Before you install a cluster in a restricted network, you must mirror the required container images into that environment. A restricted network might be one with limited network connectivity or a completely disconnected or airgapped installation. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+[IMPORTANT]
+====
+You must have access to the internet to obtain the necessary container images.
+In this procedure, you place the mirror registry on a mirror host
+that has access to both your network and the Internet. If you do not have access
+to a mirror host, use the disconnected procedure to copy images to a device you
+can move across network boundaries with.
+====
+
+[id="prerequisites_installing-mirroring-installation-images"]
+== Prerequisites
+
+* You must have a container image registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2] in the location that will host the {product-title} cluster, such as one of the following registries:
++
+--
+** link:https://www.redhat.com/en/technologies/cloud-computing/quay[Red Hat Quay]
+** link:https://jfrog.com/artifactory/[JFrog Artifactory]
+** link:https://www.sonatype.com/products/repository-oss?topnav=true[Sonatype Nexus Repository]
+** link:https://goharbor.io/[Harbor]
+--
+If you have an entitlement to Red Hat Quay, see link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html-single/manage_red_hat_quay/index#repo-mirroring-in-red-hat-quay[Introduction to repository mirroring in Red Hat Quay] for background and installation information. If you need additional assistance selecting and installing a registry, contact your sales representative or Red Hat support.
```

Comment:
I am not aware of a versionless way to link to the portal docs, in general. To my knowledge, docs.openshift.com only has the `latest` option because of the way we single source the OKD docs. @vikram-redhat, is there a way to link to `latest` on the portal?

Reply:
> is there a way to link to latest on the portal?

No. 

You can link to the latest splash page by dropping the version number ( for example, https://access.redhat.com/documentation/en-us/openshift_container_platform/) but not to a book.